src/Components/AdvancedSearch/Internal/AdvancedSearchResultDetailsList/AdvancedSearchResultDetailsList.stories.tsx

```diff
@@ -41,5 +41,6 @@ const Template: AdvancedSearchResultDetailsListStory = (args) => {
 export const Base = Template.bind({}) as AdvancedSearchResultDetailsListStory;
 Base.args = {
     twins: filteredTwins,
-    searchedProperties: cols
+    searchedProperties: cols,
+    onTwinSelection: null
```

Comment:
maybe have it log to console or something or pop an alert box with the id to see that it's working?